benchmark/README.md

```diff
@@ -0,0 +1,39 @@
+# How to Run the Benchmark tests
+
+## Prerequisites
+
+1.Install ROS 2 from binary
+
+You can download the latest binary package of ROS2 from [here](http://ci.ros2.org/view/packaging/) and follow the instructions to setup the environment.
+
+* [Linux](https://github.com/ros2/ros2/wiki/Linux-Install-Binary)
+* [macOS](https://github.com/ros2/ros2/wiki/OSX-Install-Binary)
+* [Windows](https://github.com/ros2/ros2/wiki/Windows-Install-Binary)
+
+2.Install Node.js
+
+Download from https://nodejs.org/en/
+
+3.Clone the repo `git clone https://github.com/RobotWebTools/rclnodejs.git` and `npm install`
```

Comment:
Actually, there is one more step before `npm install`: update the sub modules. I would suggest to just say `rclnodejs` installed.

Reply:
We can guide users to read https://github.com/RobotWebTools/rclnodejs#get-code